connectors/byoei.py

```diff
@@ -63,13 +65,14 @@ class Bulker:
     This class runs a coroutine that gets operations out of a `queue` and collects them to
     build and send bulk requests using a `client`
 
-    The bulk requests are controlled in several ways:
+    Arguments:
+
+    - `client` -- an `connectors.es.ESClient` instance
+    - `queue` -- an `asyncio,Queue` instance to pull docs from
```

Comment:
Similar to the other comment;
```suggestion
    - `queue` -- an instance of `asyncio,Queue` to pull docs from
```